app.js

```diff
@@ -1,14 +1,32 @@
 const express = require ('express');
-const dotenv = require ('dotenv');
+const dotenv = require ('dotenv'); //what is this
 const exphbs = require('express-handlebars');
+const morgan = require ('morgan'); //why do I need this
```

Comment:
https://www.npmjs.com/package/morgan
_HTTP request logger middleware for node.js_

That module is not mandatory. You could have a custom log for your app or even any logger at all (though it's better to have for visibility purposes (i.e. in case there is any crash)